tests/test_validate_rules.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+from copy import deepcopy
+
+import pytest
+from pydantic import ValidationError
+from pytest import raises
+
+from retasc.validator.validate_rules import validate_rule, validate_rule_dict
+
+
+def test_rule_dict_valid(valid_rule_dict):
+    try:
+        validate_rule_dict(valid_rule_dict)
+    except ValidationError:
+        assert False, "A valid rule dict should not raise ValidationError"
```

Comment:
Do you see the actual ValidationError here? It would be perhaps better to remove these `try/except` blocks and let the function just propagate the exception.